index.html

```diff
@@ -11,14 +11,14 @@
 <a href="https://github.com/opencode-mait/opencode-mait.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
 <body class="page-home">
     <section>
-        <img class="header-logo" alt="OpenCode-MAIT logo image" src="assets/img/oc_logo.jpg">
+        <img class="header-logo" alt="OpenCode-MAIT logo image" src="assets/img/oc_logo.png">
         <h1> OpenCode-MAIT</h1>
         <p>Welcome to the Free and Open Source Software Community of MAIT.</p>
     </section>
 
     <section class='social-links'>
       <ul id='listtop'>
-        <li><a title='About Us' alt="About Us" href="about">About Us</a></li>
+        <li><a title='About Us' alt="About Us" href="about.html">About Us</a></li>
```

Comment:
In the dark theme. `About Us , Github and Social` Links  backgrounds (only that portion) are white just like normal theme. You might want to change their background color in dark theme too